### PR TITLE
Update structurerandom.m

### PR DESCRIPTION
**Pull request by MARIACONCETTA**
I don't know if it is correct but i've seen that the radius of SE decrease but less than before

### CODE REVIEW

**Comment by hprasetya** [comment]
you deleted 298 lines. I assumed that's not what you wanted?

File: structurerandom.m [hunk 1]
```diff
@@ -43,7 +39,7 @@ function structurerandom(x1,x2,y1,y2)
         S.SE(i).length = lengte(S.SE(i).u,S.SE(i).v);
         
         cubed(1:Nterm) = radius^3; %determining radius extra element using murray's law: r0^3 = r1^3+r2^3
-        S.IE(nie+counterie).radius = (S.IE(nie+counterie).radius^3 - sum(cubed))^(1/3); 
+        S.IE(nie+counterie).radius = (S.IE(nie).radius^3 - sum(cubed))^(1/3);
```
Comment (hprasetya) [inline structurerandom.m:42]: This change would yield the same result, as S.IE(nie+counterie).radius is defined as S.IE(nie).radius in previous line. I don't think this is the solution.

Comment (MARIACONCETTA) [inline structurerandom.m:42]: yes I know that is not the solution....this morning i've understood it.
About lines deleted I was wrong using github,sorry.
I didn't want.

**Comment by hprasetya** [comment]
So there will be no changes made.

**Pull request closed without merging**